### ISSUE TITLE
out of date w.r.t. Silverlight and recent .NET versions?

### ISSUE THREAD

**Issue by daveyostcom** [dotnet/docs#33118]
The Silverlight mention is obsolete.

This page seems to know only about .NET Framework, nothing later.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 8cf468aa-8191-3e82-8e05-c69907e086c4
* Version Independent ID: 5b9ba335-2b0e-cafd-3409-bf1f54d16fa2
* Content: [Dynamic Language Runtime Overview - .NET Framework](https://learn.microsoft.com/en-us/dotnet/framework/reflection-and-codedom/dynamic-language-runtime-overview)
* Content Source: [docs/framework/reflection-and-codedom/dynamic-language-runtime-overview.md](https://github.com/dotnet/docs/blob/main/docs/framework/reflection-and-codedom/dynamic-language-runtime-overview.md)
* Product: **dotnet-framework**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

**Comment by adegeo** [comment]
Correct, this and the surrounding articles were all written for .NET Framework and would need to be updated and ported to .NET 6+

**Comment by daveyostcom** [comment]
Any idea when the DLR docs will be updated?

**Comment by adegeo** [comment]
I've not heard of it being a priority. @cston do you have any idea if there is a priority in modernizing this content? It should at least be reviewed for accuracy.
